Workbooks/Storage/Overview/Overview.workbook

```diff
@@ -15,7 +15,7 @@
         "version": "KqlParameterItem/1.0",
         "query": "",
         "crossComponentResources": [
-          "value::selected"
+          "{Subscription}"
```

Comment:
I don't think this part is right?  this says this very first parameter depends on another parameter called {subscription} that doesn't exist yet?

Reply:
Fixed in new PR